functions/utils/audio.py

```diff
@@ -1,38 +1,71 @@
+import wave
 from pathlib import Path
 
-import soundfile as sf
-
 
 def get_sample_rate(file: Path) -> int:
-    """Gets the current sample rate of the given audio file.
+    """Gets the current sample rate of the given wav file.
 
     Parameters:
         file: The path to the audio file.
 
     Returns:
         The sample rate of the given file.
     """
-    _, sample_rate = sf.read(file)
+    with open(file, "rb") as f:
+        wave_object = wave.open(f, "rb")
```

Comment:
Though it's often ignored, file is a keyword in Python and it's bad form to overwrite it (you'll often see me use `file_` for that reason). In this case there's a more descriptive workaround (this is just an example, pick better variable names if they occur to you):

```suggestion
def get_sample_rate(audio_file: Path) -> int:
    """Gets the current sample rate of the given wav file.
    Parameters:
        file: The path to the audio file.
    Returns:
        The sample rate of the given file.
    """
    with open(aduio_file, "rb") as wave_file:
        wave_object = wave.open(wave_file, "rb")
```

Reply:
FIxed in latest revision.